src/app.ts

```diff
@@ -21,7 +21,10 @@ import {TestRuns} from "./test-runs";
 // TestRuns.testPreProcessingChain();
 
 //Testing of WetManager
-TestRuns.testWetManager();
+//TestRuns.testWetManager();
```

Comment:
We usually only enable one of the tests to test the new feature we implemented. These test runs are not actual unit tests but more of a way to run a new feature and manually observe the behavior.